pkg/gitparse/gitparse.go

```diff
@@ -17,14 +17,6 @@ import (
 	"github.com/trufflesecurity/trufflehog/v3/pkg/context"
 )
 
-const (
```

Comment:
question: Can we actually leave the constants and then just use them in your factory function for `NewParser', but also make them private? I think they are only ever used in this file.

Ex:
```
const (
	// defaultDateFormat is the standard date format for git.
	defaultDateFormat = "Mon Jan 02 15:04:05 2006 -0700"

	// maxDiffSize is the maximum size for a diff. Larger diffs will be cut off.
	maxDiffSize = 1 * 1024 * 1024 * 1024 // 1GB
	// maxCommitSize is the maximum size for a commit. Larger commits will be cut off.
	maxCommitSize = maxDiffSize
)
```

```
// NewParser creates a GitParse config from options and sets defaults.
func NewParser(options ...Option) *Parser {
	parser := &Parser{
		dateFormat:    defaultDateFormat,
		maxDiffSize:   maxDiffSize,   // 1GB
		maxCommitSize: maxCommitSize, // 1GB
	}
	for _, option := range options {
		option(parser)
	}
	return parser
}
```